Sources/FluidAudio/Frameworks/ESpeakNG.xcframework/Info.plist

```diff
@@ -6,44 +6,42 @@
 	<array>
 		<dict>
 			<key>BinaryPath</key>
-			<string>ESpeakNG.framework/Versions/A/ESpeakNG</string>
+			<string>ESpeakNG.framework/ESpeakNG</string>
 			<key>LibraryIdentifier</key>
-			<string>macos-arm64_x86_64</string>
+			<string>ios-arm64</string>
```

Comment:
this was moslty just reodering and cleanup of the VERSIONS/A paths. its not needed